url026791d51b_dhleong_SeriesTracker_v6_tgz-pJ8-stv6_SeriesTrackerJ8/src/stv6/handlers/settings/ManageSaveHandler.java

```diff
@@ -0,0 +1,126 @@
+package stv6.handlers.settings;
+
+import java.util.Iterator;
+import java.util.LinkedList;
+import java.util.List;
+
+import stv6.Profile;
+import stv6.STServer;
+import stv6.handlers.AbstractHandler;
+import stv6.http.request.Request;
+import stv6.http.request.variables.ListVariable;
+import stv6.http.request.variables.Variable;
+import stv6.series.BasicSeries;
+import stv6.series.Series;
+import stv6.templating.Templator.Template;
+
+public class ManageSaveHandler extends AbstractHandler {
+	private final static String HANDLED_PAGE = "settings-manage-save";
+	private final static String TEMPLATE_NAME = "settings.generic.tpl";
+
+	private boolean anyNull(Object... values) {
+		for (Object o : values) {
+			if (o == null)
+				return true;
+		}
+		
+		return false;
+	}
+	
+	@Override
+	protected String getTemplatePath() {
+		return TEMPLATE_NAME;
+	}
+
+	@Override
+	protected boolean wrappedHandle(Request r, Template t) {
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The cyclomatic complexity of this method is 18. By comparison, 99% of the methods in the CodeGuru reference dataset have a lower cyclomatic complexity. This indicates the method has a high number of decisions and it can make the logic difficult to understand and test.

We recommend that you simplify this method or break it into multiple methods. For example, consider extracting the code block on lines 104-115 into a separate method.